src/AuthorizedSocketConnection.js

```diff
@@ -188,8 +188,12 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
       try {
         resultOrStream = await sourcePromise;
       } catch (err) {
-        this.subscriptionContexts.delete(id);
-        throw err;
+        if (err instanceof GraphQLError) {
```

Comment:
what a mess this is...

you know we should probably just make our own implementation of graphql :P

Reply:
i should probably upstream some fix like this

Reply:
That'd only be helpful for when the merge it in two years tho